src/content/docs/release-notes/index.mdx

```diff
@@ -176,11 +176,4 @@ To take full advantage of New Relic's latest features, enhancements, and importa
     to="/docs/release-notes/service-levels-release-notes"
   />
 
-  <TechTile
-    name="XCFramework agent"
-    icon="logo-apple-color"
-    to="/docs/release-notes/mobile-release-notes/xcframework-release-notes"
```

Comment:
Should we replace it with `/docs/release-notes/mobile-release-notes/ios-agent-release-notes`?

Reply:
No because the iOS tile is already there and with the correct link to iOS release notes. This just removes the XCFramework tile.